scripts/install-git-secrets.sh

```diff
@@ -0,0 +1,80 @@
+#! /bin/bash
```

Comment:
Wonder if these is a better solution than having this in three(?) repos.

Reply:
Can wget the script from another repo

Reply:
4 if you count the UI, should i just change all of them to grab the UI one?

Reply:
Yeah, a bit too glaring conflict with DRY and all that. 
Also, if we need to fix compatibility / improve this at some point